bigbluebutton-html5/imports/ui/components/video-provider/component.jsx

```diff
@@ -464,6 +468,197 @@ class VideoProvider extends Component {
     }
   }
 
+  customGetStats(peer, mediaStreamTrack, callback, interval) {
+    const statsState = this.state.stats;
+    const that = this;
```

Comment:
You can use `.bind` or arrow functions (better) to avoid this type of bad pattern.